src/view/ui/Saves.svelte

```diff
@@ -36,7 +36,13 @@
 
     const saves = arr
```

Comment:
Can you explicitly type `saves`?

Because the map is returning `null`, TypeScript won't infer the type properly.